sandbox-backend/src/main/java/com/exadel/sandbox/team5/service/impl/DiscountServiceImpl.java

```diff
@@ -26,6 +28,18 @@ public Discount getById(Long id) {
         return discountDAO.findAll();
     }
 
+    @Override
+    public List<Discount> getByNameOrDescriptionContaining(String searchText) {
+        List<Discount> result = new ArrayList<>();
+        if (searchText != null && !searchText.isEmpty()) {
```

Comment:
It should be improve.
For example if we have 3 discount with names, "bbb", "ccc", "ddd" and search text "aaa". Your method will still return all discounts.